test/lib/base-controller.js

```diff
@@ -329,7 +329,37 @@ describe('lib/base-controller', function () {
             }];
             controller.options.continueOnEdit = false;
             req.params.action = 'edit';
-            controller.getNextStep(req).should.contain('/confirm');
+            controller.getNextStep(req).should.equal('/confirm');
+          });
+
+          it('should return /a-base-url/target-page if completed check falsy', function () {
+            getStub.returns(['/a-base-url/target-page']);
```

Comment:
Did you remove the wrong tests?
This still has the baseUrl in the list of completed steps.